src/platform_impl/windows/event_loop.rs

```diff
@@ -1403,6 +1403,22 @@ unsafe extern "system" fn public_window_callback<T>(
 
             0
         },
+        winuser::WM_COMMAND | winuser::WM_NOTIFY => {
```

Comment:
Why only expose these two events? Why not move this to the end of the match statement and have `_ => {`?

Reply:
because... well, the wndproc is a complex mechanism. for many messages, return value is a "reply" to the system/caller.  There's the `DefWindowProc` function which generates the reply for us, but user will want to override this behavior for some messages. If we handle everything now, i think we'll need to invent a reply mechanism API now.

Maybe i was just lazy. i think i will give it a try.